backend/src/main/java/de/neuefische/backend/model/openai/ChatGptChoice.java

```diff
@@ -0,0 +1,20 @@
+package de.neuefische.backend.model.openai;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatGptChoice {
+    /**
+     {
+     "message": {
+     "content": "This is a test!"
+     }
+     }
+     */
```

Comment:
you dont want comments in your production code

Reply:
done!